src/segment-loader.js

```diff
@@ -155,6 +155,9 @@ export default videojs.extend(videojs.EventTarget, {
 
   dispose() {
     this.abort_();
+    if (this.sourceUpdater_) {
+      this.sourceUpdater_.dispose();
```

Comment:
nah, there's no benefit, at that point, you might as well just drop the if and just do
```js
this.sourceUpdater_ && this.sourceUpdater_.dispose()
```